Makefile

```diff
@@ -105,22 +120,26 @@ CFLAGS += -G 0 -non_shared -Xfullwarn -Xcpluscomm $(IINC) -nostdinc -Wab,-r4300_
 # Use relocations and abi fpr names in the dump
 OBJDUMP_FLAGS := -d -r -z -Mreg-names=32
 
+ifneq ($(OBJDUMP_BUILD), 0)
```

Comment:
There's probably a better name for this since it's not just objdump. Not sure what it is though.

Reply:
If you think of something let me know. This is still the best thing I can think of, though perhaps just `DUMP_BUILD` would be better?